plaidml/bridge/openvino/ops/deformable_convolution.cpp

```diff
@@ -0,0 +1,280 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+using namespace edsl;
+
+namespace PlaidMLPlugin {
+
+// Compute pad_before and the size of output
+std::pair<size_t, size_t> compute_padding_before_and_output_size(size_t input_size, size_t off_size, size_t filter_size,
```

Comment:
Please put all functions except `void registerDeformableConvolution()` in to 
```
namespace{
...
}
```

Reply:
like this
`namespace PlaidMLPlugin{
namespace{
...
}
void registerDeformableConvolution()
}`
or like this
`namespace{
...
}
namespace PlaidMLPlugin{
void registerDeformableConvolution()
}`

Reply:
The second one, you can check other ops as well.

Reply:
ok, got it.